code/modules/power/turbine.dm

```diff
@@ -344,7 +344,7 @@
 
 
 /obj/machinery/computer/turbine_computer/Initialize()
-	..()
+	. = ..()
 	spawn(10)
 		locate_machinery()
```

Comment:
Remove this spawn. This is a carryover from when this was still `/New()` and the atoms didnt exist physically in the world. 

Calling the locate on init should be safe now.

Reply:
Done